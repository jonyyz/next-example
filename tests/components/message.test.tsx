import { render } from "@testing-library/react";
import StoreProvider from "@/store/index.js";
// @ts-ignore
import Message from "@/components/message.tsx";

describe("Message Component", () => {
  it("should render", () => {
    // TODO: MockStoreProvider
    const { container } = render(
      <StoreProvider>
        <Message message="test" />
      </StoreProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
