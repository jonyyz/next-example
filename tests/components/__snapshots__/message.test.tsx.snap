// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component should render 1`] = `
<div>
  <div>
    <div>
      test
    </div>
    <div>
      43
    </div>
  </div>
</div>
`;
